homeassistant/components/device_tracker/owntracks.py

```diff
@@ -234,12 +234,36 @@ def async_valid_accuracy(self, message):
         return True
 
     @asyncio.coroutine
-    def async_see_beacons(self, dev_id, kwargs_param):
+    def async_see_beacons(self, hass, dev_id, kwargs_param):
         """Set active beacons to the current location."""
         kwargs = kwargs_param.copy()
+
+        # Mobile beacons should always be set to the location of the
+        # tracking device. I get the device state and make the necessary
+        # changes to kwargs.
+        device_tracker_state = hass.states.get(
+            "device_tracker.{}".format(dev_id))
+        _LOGGER.info("device tracker state: %s", device_tracker_state)
+
+        if device_tracker_state is not None:
+            try:
+                acc = device_tracker_state.attributes.get("gps_accuracy")
+                lat = device_tracker_state.attributes.get("latitude")
+                lon = device_tracker_state.attributes.get("longitude")
+                kwargs['gps_accuracy'] = acc
+                kwargs['gps'] = (lat, lon)
+            except AttributeError:
```

Comment:
I don't see a case when this will happen?

Reply:
I don't remember why I did this but I suspect it was to catch an issue I had during development. I've been running this code for several days and haven't seen the AttributeError in my logs so I'll remove it.